src/features/auth/roles.js

```diff
@@ -14,6 +14,7 @@ export const WRITE_CAMERA_REGISTRATION_ROLES =  [MANAGER];
 export const QUERY_WITH_CUSTOM_FILTER =         [MANAGER];
 export const DELETE_IMAGES =                    [MANAGER, MEMBER];
 export const MANAGE_USERS_ROLES =               [MANAGER];
+export const MANAGE_LABELS =                    [MANAGER];
```

Comment:
I've renamed the user role.

There's quite a difference between the roles in the front-end and backend: https://docs.google.com/spreadsheets/d/12NyrnElo5-LffPvxBxAd2XNkiR8kBg5IXArGzLYmfpU/edit#gid=0 The naming doesn't match up for most of them, it's hard to consolidate them because I'm not sure how and where these roles are used.